msteams-platform/tutorials/UgradeTeamsUsingYeoman.md

```diff
@@ -0,0 +1,25 @@
+# Upgrade teams using yeoman generator
+This article helps you to upgrade teams from current version to the latest version
+## Get current version of teams
+```
+ yo teams --version
+```
+## Update teams
+The yo command lists out various options ranging from creating project to upate generator , use the below command to select updtae generator
+```
+ yo
+```
+Use the arrow keys to choose Update Generators
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams/YoSelectUpdateGen.png)
+
+This should listdown all the generators available , you should select a specific generator by using space bar to select and deselect teams from the available options
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams//UseSpaceToSelectGenerators.png)
+
+It takes a while to install teams, may be a few seconds to minutes
+
+Once the installation is complete , you can use the below command to check the version
+
+```
+yo teams --version
+```
```

Comment:
Same comment as for the previous command: Will we need to call out the language against the ticks?
For example, mention json after the (```) ?
```json
yo teams --version
```